betel/classifier_data_set_builder.py

```diff
@@ -0,0 +1,67 @@
+import shutil
+import pathlib
+import pandas as pd
+from betel import utils
+from betel import info_files_helpers
+from betel import data_set_builder
+
+
+class ClassifierDataSetBuilder(data_set_builder.DataSetBuilder):
+    """A class for splitting classifier data into train-validation-test sets."""
+
+    def __init__(self, input_dir: pathlib.Path, storage_dir: pathlib.Path,
+                 split_ratio: (float, float, float) = (0.7, 0.15, 0.15),
+                 classes: [str] = None):
+        """Constructor.
+
+        :param input_dir: data to be split (output of the scraper,
+        meaning a directory containing the input icons and a csv file
+        describing the whole data set)
+        :param storage_dir: storage directory for split data sets
+        :param split_ratio: the ratio for train-validation-test data sets
+        :param classes: the classes desired for the classifier (should be
```

Comment:
If classes is None I would expect it to default to the empty array. Instead it defaults to all classes.
You should make that clear in the comment and also change the typehint to Optional[List[str]]